README.md

```diff
@@ -30,16 +31,18 @@ you are interested in battle-tested, production-ready Terraform code, check out
 
 ### Pre-requisites
 
-1. Install [Terraform](https://www.terraform.io/) version `1.1.4` and
+1. Install [OpenTofu](https://opentofu.org/) version `1.6.0` or newer and
    [Terragrunt](https://github.com/gruntwork-io/terragrunt) version `v0.36.0` or newer.
```

Comment:
```suggestion
   [Terragrunt](https://github.com/gruntwork-io/terragrunt) version `v0.52.0` or newer.
```

based on the [Terragrunt compatibility table](https://terragrunt.gruntwork.io/docs/getting-started/supported-versions/#supported-opentofu-versions).